www-apps/tt-rss/tt-rss-99999999.ebuild

```diff
@@ -50,9 +49,39 @@ DEPEND="
 
 need_httpd_cgi # From webapp.eclass
 
-PATCHES=(
-	"${FILESDIR}"/${PN}-no-chmod.patch
-)
+pkg_pretend() {
+	if has_version www-apps/tt-rss[mysqli]; then
+		ewarn
+		ewarn "You are currently using tt-rss with mysql backend."
+		ewarn
+		ewarn "THIS IS NOT SUPPORTED ANYMORE."
+		ewarn
+		ewarn "Since 2025-04-14 tt-rss has dropped support for MySQL."
+		ewarn
+		ewarn "To upgrade, you need to migrate to PostgreSQL first."
+		ewarn
+		ewarn "Migrating between different tt-rss versions might work but is not recommended."
+		ewarn "It's recommended to switch to =www-apps/tt-rss-20250414 for migration."
+		ewarn
+		ewarn "Export/Import articles could be done with an official plugin:"
+		ewarn "ttrss-data-migration"
+		ewarn "For plugin installation and export/import, see:"
+		ewarn "https://gitlab.tt-rss.org/tt-rss/plugins/ttrss-data-migration"
+		ewarn
+		ewarn "Example of migration steps:"
+		ewarn "0. Backup !"
+		ewarn "1. Setup PostgreSQL backend"
```

Comment:
By this, do you just mean setup PostgreSQL itself, but don't point tt-rss to it yet? I think the word "backend" maybe confuses it.

Reply:
ok. Changed with package name and without "backend".